lib/puppet/functions/hiera_common.rb

```diff
@@ -0,0 +1,67 @@
+require 'hiera_puppet'
+
+# Provides the block that defines the hiera, hiera_array, and hiera_hash functions. The only difference
+# between those functions is the merge type (:priority, :array, or :hash) passed to the Hiera lookup. The
+# definition contained here derives the merge type from the name of the function.
+#
+class Puppet::Functions::HieraCommon
```

Comment:
We have no lib/puppet/hiera namespace. We do have a lib/hiera namespace though and we also have a HieraPuppet class in the /lib namespace. Is it OK if I place it in the lib/hiera namespace rather than creating a third namespace for hiera related stuff?